aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js

```diff
@@ -72,107 +68,60 @@ const allData = [
-        <FiltersProvider>
-          <Box gap="medium">
-            <Header pad={{ top: 'medium' }}>
-              <Box gap="xsmall" fill="horizontal">
-                <Heading id="users-heading" level={2} margin="none">
-                  Users
-                </Heading>
-                <Box direction="row" justify="between" align="start" wrap>
-                  <FilterControls
-                    // Table column configuration should be grouped on right
-                    // side with other actions to separate it from filtering.
-                    // If grouped to the left with the Filter control, it
-                    // becomes confusing what the "Clear filters" button will
-                    // do with regards to any column configurations that have
-                    // been applied.
-                    configure={
-                      <DropButton
-                        a11yTitle="Configure columns button"
-                        icon={<Splits />}
-                        kind="toolbar"
-                        dropAlign={{ top: 'bottom', right: 'right' }}
-                        onClose={() => setOpen(false)}
-                        onOpen={() => setOpen(true)}
-                        dropContent={
-                          <ColumnSettings
-                            columns={COLUMNS}
-                            visibleColumns={visibleColumns}
-                            setVisibleColumns={setVisibleColumns}
-                            open={open}
-                          />
-                        }
-                        tip="Configure columns"
-                      />
-                    }
-                    actions={<Menu kind="toolbar" label="Actions" items={[]} />}
-                    data={allData}
-                    filters={filtersConfig}
-                    searchFilter={{ placeholder: 'Search' }}
-                  />
-                </Box>
-              </Box>
-            </Header>
-            <Results columns={visibleColumns} />
-          </Box>
-        </FiltersProvider>
-      </PageContent>
-    </Page>
-  );
-};
+export const TableCustomizationExample = () => (
+  <Page background="background" fill>
+    <PageContent>
+      <Box gap="medium">
+        <Header pad={{ top: 'medium' }}>
+          <Heading id="users-heading" level={2} margin="none">
+            Users
+          </Heading>
+        </Header>
+        <Results />
+      </Box>
+    </PageContent>
+  </Page>
+);
 
-const Results = ({ columns }) => {
+const Results = () => {
   const [select, setSelect] = useState([]);
-  const { filteredResults } = useFilters();
 
   return (
-    <Box fill overflow="auto">
-      <DataTable
-        aria-describedby="users-heading"
-        data={filteredResults}
-        background="background"
-        columns={columns}
-        select={select}
-        onSelect={setSelect}
-        pin
-      />
-    </Box>
+    <Data data={allData} updateOn="submit" flex>
```

Comment:
In order for the filters layer to also render the custom labels used in DataTableColumns, should you pass the following? I haven't tried locally so you'll need to confirm.
```suggestion
    <Data data={allData} updateOn="submit" properties={columnsConfig} flex>
```

Reply:
I'm also wondering if a better name for columns config would be `options` or `properties`.

Reply:
The filters column doesn't render custom labels, but I'll have to look at changing properties because I'm getting this error: 
<img width="961" alt="image" src="https://user-images.githubusercontent.com/23144209/218839475-997ce5f5-4a62-4513-8854-9fcc6a255d9e.png">